README.md

```diff
@@ -1,139 +1,95 @@
+## Eligibility
 
-Note: `linked-addon.xpi` is a symbolic link to the extension's true XPI, which is named based on the study's unique addon ID specified in `package.json`.
+Users are ineligible for the study if:
+* They have already changed their Tracking Protection setting from the default.
 
+## Treatment Branches
 
-## About This Study
+1. `"control"`: The user receives no messaging or prompts, and Tracking Protection is kept at its default setting (ON in private windows only).
+2. `"pseudo-control"`: The user receives no messaging or prompts, and Tracking Protection is set to ON globally.
+3. `"fast"`: The user receives messaging about how Tracking Protection saves them time. Tracking Protection is set to ON globally.
+4. `"private"`: The user receives messaging about how Tracking Protection protects their privacy. Tracking Protection is set to ON globally. 
 
-[Study PHD](https://docs.google.com/document/d/1OJq1EDMNqydNoR8Ui_916Uzq7SHq9rl0bfZTmO5dC3g/)
+## User Flow
 
-Goal:  Determine the optimal messaging for tracking protection in Firefox during new user onboarding.
+1. (Fast and Private branches only) For the first webpage the user visits with blocked resources, a pageAction icon will appear, displaying an introductory panel.
+2. (Fast and Private branches only) For every subsequent webpage the user visits with blocked resources, the pageAction badge will display:
+- The amount of time saved in seconds (Fast)
+- The number of blocked resources (Private)
+If the user clicks on the pageAction icon, a panel will open with more information.
+3. (Fast and Private branches only) When the user visits `about:newtab` or `about:home`, a message on the page will display per-session totals for:
+- number of blocked resources AND
+- time saved per session (Fast) OR
+- number of tracking companies blocked (Private)
+4. (All branches) If the study is still running after two weeks, the study will end, restoring Tracking Protection to its default setting and opening a new tab to a survey.
 
+### Notes
+* (Fast and Private branches only) If the user disables Tracking Protection from either the introduction panel or the subsequent pageAction panel, the study will end.
+* (All branches) If at any time during the study the user modifies their Tracking Protection setting, the study will end.
 
-## User Experience / Functionality
+## Testing Preferences
+The following preferences can be set to customize the study behavior for testing purposes.
 
-**Under Construction** TODO bdanforth: update
+<dl>
+  <dt><code>extensions.tracking_protection_messaging_study.variation_override</code></dt>
+  <dd>The treatment to use. Set this to a value from the Treatment Branches section to force the add-on to show you that treatment. You must set this preference before installing the study (default: random).</dd>
 
+  <dt><code>extensions.tracking_protection_messaging_study.duration</code></dt>
```

Comment:
Pref should be `"extensions.tracking_protection_messaging_study.duration_override"` per #46 .